lib/mongoid_session_store.rb

```diff
@@ -0,0 +1,24 @@
+require 'securerandom'
+$:.unshift File.dirname(__FILE__)
+
+require 'mongoid_session_store/store'
+
+module MongoidSessionStore
+  autoload :VERSION, 'mongoid_session_store/version'
+
+  def self.collection_name=(name)
+    @collection_name = name
+
+    MongoidSessionStore::Session.store_in collection: name
```

Comment:
The load and configuration process is now straightforward, we only support Mongoid.